secure_message/repository/modifier.py

```diff
@@ -97,3 +99,45 @@ def mark_message_as_read(message, user):
                 raise InternalServerError(description="Error adding read information to message")
         Modifier.remove_label(unread, message, user)
         return True
+
+    @staticmethod
+    def close_conversation(metadata, user):
+        bound_logger = logger.bind(conversation_id=metadata.id, user_id=user.user_uuid)
+        bound_logger.info("Getting user details")
+
+        user_details = internal_user_service.get_user_details(user.user_uuid)
+        bound_logger.info("Successfully retrieved user details")
+
+        try:
+            bound_logger.info("Closing conversation")
```

Comment:
what about add conversation id in the log file so at leats we have some trace of it in case ?

Reply:
It is, that's what the bound logger is doing.